SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialWorkerFlags.h

```diff
@@ -20,22 +19,21 @@ class SPATIALGDK_API USpatialWorkerFlags : public UBlueprintFunctionLibrary
 	 * @param OutValue - Value of worker flag
 	 * @return - If worker flag was found.
 	 */
-	UFUNCTION(BlueprintCallable, Category="SpatialOS")
-	static bool GetWorkerFlag(const FString& Name, FString& OutValue);
-	
-	static FOnWorkerFlagsUpdated& GetOnWorkerFlagsUpdated();
+	bool GetWorkerFlag(const FString& Name, FString& OutValue);
```

Comment:
Is this function intended to be accessible from blueprint?

Reply:
I added the` bool USpatialStatics::GetWorkerFlag(const UObject* WorldContext, const FString& InString, FString& OutString)` function that is accessible from blueprint so I do not see why this one needs to be accessible too?

Reply:
Yes, my bad didn't read properly;)